src/app/visit/treatment.cpp

```diff
@@ -23,3 +23,16 @@ std::string toString(Treatment dental_treatment)
 
     return searched_pair != DentalTreatments.end() ? searched_pair->second : "other treatment type";
 }
+
+Treatment toEnumTreatment(const std::string& dental_treatment)
+{
+    for (const auto& [key, value] : DentalTreatments)
+    {
+        if (value == dental_treatment)
+        {
+            return key;
+        }
+    }
```

Comment:
should work. Optionaly to consider: using std::string::find.
I don't know, perhaps string::find optimises searching somehow.